x-pack/plugins/apm/public/components/app/correlations/error_correlations.tsx

```diff
@@ -130,7 +126,9 @@ export function ErrorCorrelations({ onClose }: Props) {
   );
 
   const trackApmEvent = useUiTracker({ app: 'apm' });
-  trackApmEvent({ metric: 'view_errors_correlations' });
+  trackApmEvent({ metric: 'view_failed_transactions' });
+
+  const onFilter = () => {};
```

Comment:
TODO?

Reply:
This whole component will be replaced by the work done in #108441 that's why I didn't fix the previous flyout `onClose` for the tabs for this PR.